### ISSUE TITLE
[Feature] Preferred speed and Reset speed

### ISSUE THREAD

**Issue by pintassilgo** [Andrews54757/FastStream#237]
I'm a long time [Video Speed Controller](https://addons.mozilla.org/en-US/firefox/addon/videospeed/) user, a very popular extension with >3kk users in Chrome. It doesn't work with FastStream.

Actually, there are only two missing commands in FastStream, both are useful to me:

- A shortcut to a preferred speed (number set by user in extension options). So, for instance, to use 3× speed I'd just press a key instead of having to press the `Increase Playback Rate` multiple times depending on step size.

- Shortcut to reset speed, toggling between 1× and current speed. Example: to alternate between 3×, which is my preferred speed, and 1×. But should not be tied to preferred speed, because there are some videos I need to watch in not so fast speed, like 2× instead of my preferred 3×, and I still want to be able to quickly alternate between this custom speed and 1× while watching the video. As another example, I might want to toggle between slow motion (0.1×) and 1×... So, as said, `Reset speed` should switch between current speed (whatever it is) and 1×.

**Comment by Andrews54757** [comment]
Will do with settings rewrite

**Comment by Andrews54757** [comment]
Duplicate of #214
